src/components/SelectionPills/SelectionPill/docs/SelectionPill.md

```diff
@@ -0,0 +1,38 @@
+**`SelectionPill` example:**
+
```

Comment:
Below paragraph is good since it describes how to use the component, right above can you add a quick tl;dr for what `SelectionPill` is and maybe how it differs from navigation pills?